{"name": "test_delete_pet_error", "status": "passed", "attachments": [{"name": "log", "source": "d646a220-46bf-4fa5-8502-a2ffe07ee58f-attachment.txt", "type": "text/plain"}], "start": 1727372359386, "stop": 1727372359912, "uuid": "62f24121-022a-4d24-8cf5-7e68d111bfdc", "historyId": "4ddc46705d47961f73b9cf4d121bfebd", "testCaseId": "4ddc46705d47961f73b9cf4d121bfebd", "fullName": "tests.test_pet#test_delete_pet_error", "labels": [{"name": "feature", "value": "API_PETSTORE"}, {"name": "story", "value": "Проверка, что удаление несуществующего питомца возвращает ошибку"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_pet"}, {"name": "host", "value": "DESKTOP-PF72GEU"}, {"name": "thread", "value": "26252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_pet"}]}